libbeat/cmd/instance/beat.go

```diff
@@ -348,11 +346,14 @@ func NewBeatReceiver(settings Settings, receiverConfig map[string]interface{}, u
 		return nil, fmt.Errorf("error unpacking beats logging config: %w\n%v", err, b.Config.Logging)
 	}
 
-	if err := logp.ConfigureWithCore(logpConfig, core); err != nil {
+	b.Info.Logger, err = logp.ConfigureWithCoreLocal(logpConfig, core)
+	if err != nil {
 		return nil, fmt.Errorf("error configuring beats logp: %w", err)
 	}
+	// extracting it here for ease of use
+	logger := b.Info.Logger
```

Comment:
Despite this shortcut variable we still use a mixture of `logger` and `b.Info.Logger` below. Should we make it more consistent and use only `logger`? Otherwise, there is no point in this variable if we don't use it for everything.